package.json

```diff
@@ -6,9 +6,10 @@
   "typings": "dist/main.d.ts",
   "scripts": {
     "dopublish": "npm run type-check-docs && npm run package && npm run publish-code-verification && npm publish",
-    "lint": "tslint src/**/*.ts scripts/**/*.ts",
+    "lint": "tslint --project tsconfig.json",
```

Comment:
Could you switch this back? I'd like to lint the scripts folder as well, which isn't specified in tsconfig.json.